### DISCUSSION TITLE
.Net versions supported

### DISCUSSION THREAD

**Discussion by ZavalichiR** [danielpalme/ReportGenerator#419] [Q&A] [answered]
Hello!
Which are the latest .Net versions suported? (.Net x, .Net core and .Net framework)
Are you planning to support .Net 6 version?

**Comment by danielpalme** [comment, marked as answer]
Currently .NET 4.7 or .NET Core 2.x/3.x/5.x. are supported.
Support for .NET 6 will be added as soon as it is published.
